[Bug]: 2 tests fail: rustup_proxy_build and parse_target_triple_build

### Current Behavior

For rustup_proxy_build:
```
CMake Error at CMakeLists.txt:24 (message):
  `/usr/local/bin/rustc --version` output does not match rustup: rustc 1.83.0
  (90b35a623 2024-11-26) (built from a source tarball)
```

For parse_target_triple_build:
```
-- Parsed Target triple: arch: xtensa, vendor: esp32s3, OS: none, env: elf
-- Configuring done (5.7s)
-- Generating done (0.0s)
-- Build files have been written to: /usr/ports/devel/corrosion/work/.build/test/parse_target_triple/build-parse_target_triple
'/usr/local/bin/cmake' '--build' '/usr/ports/devel/corrosion/work/.build/test/parse_target_triple/build-parse_target_triple'
ninja: no work to do.
<end of output>
Test time =   5.90 sec
----------------------------------------------------------
Test Fail Reason:
Error regular expression found in output. Regex=[CMake Warning]
"parse_target_triple_build" end time: Jan 09 15:36 PST
"parse_target_triple_build" time elapsed: 00:00:05
----------------------------------------------------------
```

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Environment

```markdown
- OS: FreeBSD 14.2
- CMake:3.31.3
- CMake Generator: ninja
```


### CMake configure log with Debug log-level

_No response_

### CMake Build step log

_No response_

--- a/test/find_rust/CMakeLists.txt
+++ b/test/find_rust/CMakeLists.txt
@@ -1,3 +1,7 @@
 corrosion_tests_add_test(find_rust "")
 corrosion_tests_add_test(rustup_proxy "")
 
+find_program(rustup rustup)
+if(NOT rustup)
+    set_tests_properties(rustup_proxy_build rustup_proxy_cleanup PROPERTIES DISABLED 1)
+endif()

--- a/test/parse_target_triple/CMakeLists.txt
+++ b/test/parse_target_triple/CMakeLists.txt
@@ -2,9 +2,9 @@ corrosion_tests_add_test(parse_target_triple "")
 corrosion_tests_add_test(parse_target_triple_should_fail "")
 
 set_tests_properties("parse_target_triple_build" PROPERTIES FAIL_REGULAR_EXPRESSION
-        "CMake Warning"
+         "CMake Warning at [^\r\n]*FindRust\.cmake:.*(\r)?\n[ \t]*Failed to parse target-triple `"
         )
 
 set_tests_properties("parse_target_triple_should_fail_build" PROPERTIES PASS_REGULAR_EXPRESSION
-    "CMake Warning"
+         "CMake Warning at [^\r\n]*FindRust\.cmake:.*(\r)?\n[ \t]*Failed to parse target-triple `"
     )

